pkgs/applications/networking/p2p/transmission/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, callPackage }:
+
+let
+  self = rec {
```

Comment:
Is it possible to use `finalAttrs` rather than `rec` here?

Reply:
probably if i make it a separate package as suggested by @emilazy

Reply:
ah so i just tried that, that blows up because the builtinput then is screwy. will have to keep the let/rec for now -.-